src/models/Env.ts

```
@@ -0,0 +1,58 @@
+  /** Default value returned when the environment variable is not set */
+  defaultValue?: T;
+  /** How should the string value coming from process.env be cast */
+  cast?: EnvCastType;
+}
+
+function castValue(raw: string | undefined, cast: EnvCastType | undefined) {
+  if (raw === undefined) return undefined;
+
+  switch (cast) {
+    case "number":
+      const n = parseFloat(raw);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)